controls/1_1_filesystem_configuration.rb

```diff
@@ -237,6 +237,8 @@
   end
 end
 
+# There is a mistake in the official CIS DIL documentaion 1.1.10-1.1.14 are
```

Comment:
Should we remove them? I think there is no value in adding CIS misstakes in this profile. We could point this out in the README.

Reply:
Agreed, these can be left out. It's acknowledged as a bug by the CIS maintainers, however unresolved for about 8 months now.